client/src/components/Ratings&Reviews/Main.jsx

```diff
@@ -37,26 +70,38 @@ class Main extends React.Component {
     });
   }
 
-  getMetaData() {
-    const { productId } = this.state;
-    // GET request for reviews for the specific product
+  getNewestReviews() {
+    const { productId } = this.props;
     $.ajax({
-      url: `http://localhost:8080/atelier/reviews/meta?product_id=${productId}`,
+      url: `/atelier/reviews?product_id=${productId}&count=30&sort=newest`,
       type: 'GET',
       success: (data) => {
         this.setState({
-          ratingData: data,
+          productReviews: data,
+        });
+      },
+    });
+  }
+
+  getHelpfulReviews() {
+    const { productId } = this.props;
+    $.ajax({
+      url: `/atelier/reviews?product_id=${productId}&count=30&sort=helpful`,
+      type: 'GET',
+      success: (data) => {
+        this.setState({
+          productReviews: data,
         });
       },
     });
   }
```

Comment:
Instead of having unique functions to pull reviews for each filter option (e.g., relevant, helpful, newest), could combine them into one function that gets reviews based on a sorting variable passed into the function, which could be added to the url string the same way you've added the productId using a template literal.